src/test/java/Lesson15/OfficeDaoImplIntegrationTest.java

```diff
@@ -24,4 +34,23 @@ public void testGetByRegion() throws SQLException {
         System.out.println(offices.size());
         assertEquals(offices.size(), 3);
     }
+
+    @Test
+    public void test1InsertOffice() throws SQLException {
+        assertTrue(officeDao.insertOffice(office1));
+    }
+
+    @Test
+    public void test2UpdateOffice() throws SQLException {
+        assertTrue(officeDao.updateOffice(office2));
+
+    }
+
+    @Test
+    public void test3DeleteOffice() throws SQLException {
+        assertTrue(officeDao.deleteOffice(office2.getOfficeId()));
+        Set<Office> offices = officeDao.getAllOffices();
+        assertFalse(offices.contains(office1));
```

Comment:
Your office1 and office2 there are same object in DB. Your is possible, but may hard understand